src/components/userpanel/StampRedeem.tsx

```diff
@@ -18,27 +25,176 @@ const ShippingText = styled.span`
   justify-content: center;
   align-items: center;
   display: flex;
-  font-size: 2vh;
+  font-size: 2.425vh;
   font-weight: 800;
   color: white;
   text-shadow: -2px 2px 0 #000, 2px 2px 0 #000, 2px -2px 0 #000,
     -2px -2px 0 #000;
 `;
 
+const ShippingHighlight = styled.span`
+  margin: 0;
+  font-size: 3rem;
+  font-weight: 800;
+  color: #ead182;
+  text-shadow: -2px 2px 0 #000, 2px 2px 0 #000, 2px -2px 0 #000,
+    -2px -2px 0 #000;
+`;
+
 const StampRedeem = () => {
-  const [disabled] = useState(true);
+  const walletAddress = useSelector(selectGlobalAccount);
+
+  const url = process.env.REACT_APP_STORE_URL;
+  const aikoAPI = process.env.REACT_APP_EXPRESS_SERVER_URL;
+  const [disabled, setDisabled] = useState(false);
+  const [encryptedObject, setEncryptedObject] = useState("");
+
+  const [key] = useState("aikoaikoaiko");
+
+  const stampsHeld = useSelector(selectStampsHeld);
+
+  const fetchDiscount = async (existsId: number) => {
+    try {
+      const response = await fetch(`${aikoAPI}/get-discount/${existsId}`);
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      const data = await response.json();
+      // setCode(data.discount_codes[0].code);
```

Comment:
Can probably remove the comment and logging

Reply:
Done! Thank you!